src/libData/AccountData/Transaction.cpp

```diff
@@ -202,7 +211,9 @@ bool Transaction::IsSignedECDSA() const {
   sigString = sigString.substr(2);
   pubKeyStr = pubKeyStr.substr(2);
 
-  return VerifyEcdsaSecp256k1("", sigString, pubKeyStr);
+  auto hash = GetOriginalHash(GetCoreInfo(), ETH_CHAINID_INT);
```

Comment:
If we don't modify `hash` in `VerifyEcdsaSecp256k1` then it should be marked as const.

Reply:
👍